### PR TITLE
Ccx Bid Adapter: adomain support

### PR DESCRIPTION
**Pull request by mjaworskiccx**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/ccxBidAdapter.js [hunk 1]
```diff
@@ -117,7 +117,8 @@ function _buildResponse (bid, currency, ttl) {
     creativeId: bid.crid,
     netRevenue: false,
     ttl: ttl,
-    currency: currency
+    currency: currency,
+    advertiserDomains: bid.adomain || []
```
Comment (patmmccann) [inline modules/ccxBidAdapter.js:121]: Meta.advertiserDomains please

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by mjaworskiccx** [comment]
added

**Comment by robertrmartinez** [comment]
please add a unit test!

**Comment by patmmccann** [comment]
@mjaworskiccx here is an example of a unit test for this change: https://github.com/prebid/Prebid.js/pull/6495/files

**Comment by mjaworskiccx** [comment]
@patmmccann thx, unit test added

**Comment by robertrmartinez** [comment]
Looks good, assigning to @patmmccann for re-review

**Pull request merged**